opensearch_dashboards.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "securityDashboards",
-  "version": "3.0.0.0",
-  "opensearchDashboardsVersion": "3.0.0",
+  "version": "2.6.0.0",
```

Comment:
In main these should stay as 3.0.0 and 3.0.0.0, when backporting these versions will reflect the values on their respective branches

Reply:
Of course thank you @cwperks, this is now as expected.